server/db/040-deprecate-one-signal.sql

```diff
@@ -0,0 +1,35 @@
+DO $migration$
+    DECLARE migrationId INT;
+    DECLARE lastSuccessfulMigrationId INT;
+BEGIN
+    -- The migration ID of this file
+    migrationId := 40;
+
+    -- Get the migration ID of the last file to be successfully run
+    SELECT MAX(id) INTO lastSuccessfulMigrationId
+    FROM migrations;
+
+    -- Only execute this script if its migration ID is next after the last successful migration ID
+    IF migrationId - lastSuccessfulMigrationId = 1 THEN
+        -- Remove columns alert_api_key and responder_push_id from clients
+        ALTER TABLE clients DROP COLUMN alert_api_key;
+        ALTER TABLE clients DROP COLUMN responder_push_id;
+
+	-- Where responder_phone_numbers is NULL, set it to an empty array
+	UPDATE clients
+	SET responder_phone_numbers = '{}'
+	WHERE responder_phone_numbers IS NULL;
```

Comment:
Similar to Sensors, I suggest disabling the trigger while doing this update so that the `clients.updated_at` isn't changed.